src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const logo = document.querySelector('.logo');
+
+const message = document.createElement('div');
+const errorMessage = document.createElement('div');
+
+message.classList = 'message';
+message.innerText = 'Promise was resolved!';
+
+errorMessage.classList = 'message error-message';
+errorMessage.innerText = 'Promise was rejected!';
+
+function createPromise() {
```

Comment:
The task requires creating two promises, one resolved and one rejected. The current implementation creates two promises that resolve on clicking the logo or reject after a 3-second timeout. Consider creating two separate promises as specified in the task instructions.